src/dialogs/PaymentErrorDialog.vue

```diff
@@ -28,19 +28,18 @@
         <!-- display errors -->
         <div class="font-15 mb-4" v-if="numErrors > 0">
           <p>We were unable to process your payment due to the following errors:</p>
-          <ul>
-            <li v-for="(error, index) in errors" :key="index">
-              {{error.message}}
-            </li>
-          </ul>
+          <div v-for="(error, index) in errors" :key="index" class="d-flex">
+            <span class="flex-shrink-0 bullet"><v-icon>mdi-circle-medium</v-icon></span>
+            <span class="flex-grow-1" v-html="error.message" />
+          </div>
         </div>
 
         <!-- display warnings-->
         <div class="font-15 mb-4" v-if="numWarnings > 0">
           <p>Please note the following warnings:</p>
           <ul>
             <li v-for="(warning, index) in warnings" :key="index">
-              {{warning.message}}
+              {{ warning.warning }}
```

Comment:
I checked with the test unit, and the key is warning instead of message, so I changed it to warning.warning.

Reply:
Are you sure the unit test isn't wrong?

Reply:
Yes. The unit test is fine.

Reply:
Hmmm. It’s fine but it could be wrong. I’ll check again.

Reply:
Yes, that's what I meant 😁